src/azure-cli-core/azure/cli/core/commands/arm.py

```diff
@@ -631,7 +666,11 @@ def _update_instance(instance, part, path):
                 show_options(instance, part, path)
 
             if '=' in index.group(1):
-                key, value = index.group(1).split('=')
+                key, value = index.group(1).split('=', 1)
```

Comment:
It's not needed because you can't nest the square brackets so the performance optimization holds here.